### DISCUSSION TITLE
Menu not showing the tenant profile page

### DISCUSSION THREAD

**Discussion by micraux** [filamentphp/filament#10900] [Help]
Hello,

I am trying to create a tenant profile page, following this section in the docs:
[https://filamentphp.com/docs/3.x/panels/tenancy#adding-a-tenant-profile-page](https://filamentphp.com/docs/3.x/panels/tenancy#adding-a-tenant-profile-page)

But I can't see a link to this page in the main menu...

Did I miss something? Do I have to declare it somewhere?

**Comment by leandrocfe** [comment]
Is the username a link?
![image](https://github.com/filamentphp/filament/assets/3833889/378cb622-b67d-48e9-8768-572a68ce2f90)

**Reply by micraux** [reply]
No...

**Reply by micraux** [reply]
@leandrocfe,

Do you mean the username should be a link pointing to the **_tenant_** profile page I am trying to create?

Please note that I am not speaking about any **_user_** profile page...

**Reply by micraux** [reply]
Well, not important for now, I'll retry when I have time.
I close this discussion.
